### DISCUSSION TITLE
getting error on form submissions

### DISCUSSION THREAD

**Discussion by thenoahhein** [payloadcms/payload#4635] [Q&A] [answered]
hey all, was wondering as I couldn't tell from the documentation or guide on the form-builder plugin. Are emails mandatory? I've got a simple form that takes a user's email and adds it to the database. that's working great. But upon submission, I get an error in my logs `err: "Error while sending one or more emails in form submission id: undefined."`

I don't know how the form submission Id could be undefined given that the ID is successfully generated and passed to the backend as the ID is there. But really just confused as I don't h ave any emails set up on that particular form, nor do I have email enabled in the `payload.config` file.

do I need to set up email for this? I can't imagine that's a sane default and everything else has made a fair amount of sense so far. Just wanted some clarification on this!

**Comment by thenoahhein** [comment, marked as answer]
answer: I was passing the id from the `block` in my component, not the form id.

**Reply by cmengu** [reply]
Hey ngeingit, facing the same issue here! I couldn't understand what you meant by "passing the id from the block in my component, not the form id", could you explain more in depth?
